islands/serviceMap.tsx

```diff
@@ -144,7 +152,9 @@ export default function ServiceMapComponent(
       const updated = updateNode(nodes, opUpdateSignal.value);
       setNodes(updated);
     }
+  });
 
+  useSignalEffect(() => {
```

Comment:
This was actually a fairly significant unrelated bug. We had one signal effect handler to track changes on both opModal and the service map...so we conflated those two side effects.